Validation/RecoTrack/python/TrackValidation_cff.py

```diff
@@ -674,3 +674,21 @@ def _getMVASelectors(postfix):
     tracksValidationTruth +
     trackValidatorLite
 )
+
+## customization for timing
+from Configuration.Eras.Modifier_phase2_timing_layer_cff import phase2_timing_layer
+phase2_timing_layer.toModify( generalTracksFromPV, 
+                              vertexTag = cms.InputTag('offlinePrimaryVertices4D'),
+                              timesTag  = cms.InputTag('trackTimeValueMapProducer:generalTracksConfigurableFlatResolutionModel'), 
+                              timeResosTag = cms.InputTag('trackTimeValueMapProducer:generalTracksConfigurableFlatResolutionModelResolution'), 
+                              nSigmaDtVertex = cms.double(3) )
+phase2_timing_layer.toModify( trackValidatorStandalone,
+                              label_vertex = cms.untracked.InputTag('offlinePrimaryVertices4D') )
+phase2_timing_layer.toModify( trackValidatorFromPVStandalone,
+                              label_vertex = cms.untracked.InputTag('offlinePrimaryVertices4D') )
+phase2_timing_layer.toModify( trackValidatorFromPVAllTPStandalone,
+                              label_vertex = cms.untracked.InputTag('offlinePrimaryVertices4D') )
+phase2_timing_layer.toModify( trackValidatorConversionStandalone,
+                              label_vertex = cms.untracked.InputTag('offlinePrimaryVertices4D') )
+phase2_timing_layer.toModify( trackValidatorGsfTracks,
+                              label_vertex = cms.untracked.InputTag('offlinePrimaryVertices4D') )
```

Comment:
Is there a reason to customize only some of the MTV instances?

And what about the vertex validation?

Reply:
Yep, we should probably get that too, but I don't want to add anything else to this PR at the moment. We can fix it in a follow up (this era is never used is real life anyway).

Reply:
Subsequent PR is fine for me. I'd just want to get the file eventually consistent (or have comments why consistency is not wanted for some case).

Reply:
actually, if this does not cover the PV validation, then I will at least need the snippet to rerun the tests.
I have naively assumed that the last iteration had it all.
The 4D PV validation for this PR has been an outstanding item for almost a month.

Reply:
This file all about track validation, PV validation is configured here
[Validation/RecoVertex/python/PrimaryVertexAnalyzer4PUSlimmed_cfi.py](https://github.com/lgray/cmssw/blob/efbe344b8290dadec925c0c5c3e3f9329ecf5e9c/Validation/RecoVertex/python/PrimaryVertexAnalyzer4PUSlimmed_cfi.py)